packages/x-date-pickers/src/internals/hooks/usePicker/usePickerValue.types.ts

```diff
@@ -344,7 +340,31 @@ export interface UsePickerValueContextValue {
    */
   setOpen: React.Dispatch<React.SetStateAction<boolean>>;
   /**
-   * `true` if the picker is open, `false` otherwise.
+   * Set the current value of the picker to be empty.
+   * The value will be `null` on single pickers and `[null, null]` on range pickers.
    */
-  open: boolean;
+  clearValue: () => void;
```

Comment:
The main thing I would like to discuss is the naming of those methods.

I tried to be consistent with `setOpen` (vs `onOpen` / `onClose`).